v2/third_party/go/pkgsite/README.md

```diff
@@ -0,0 +1,9 @@
+# go/pkgsite/source
+
+Vendored from <https://go.googlesource.com/pkgsite/+/ef8047b111963f61f5a0b3ae5b464cdc18dc5f74/internal/source>,
+because the source package is internal, and there's no plan to move it out anytime soon: <https://github.com/golang/go/issues/40477#issuecomment-868532845>.
+
+Local modifications:
+
+* Tests are not vendored.
```

Comment:
For completeness, let's include the tests if it's not that much trouble.

Reply:
Makes sense to me, I'll do so.